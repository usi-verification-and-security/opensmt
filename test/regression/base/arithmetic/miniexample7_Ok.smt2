(set-logic QF_LIA)
(declare-fun x () Int)
(declare-fun y () Int)
(define-fun uninterp_mul ((a Int) (b Int)) Int (div a b))
(assert (= (uninterp_mul 10 y) x))
(check-sat)
(exit)
