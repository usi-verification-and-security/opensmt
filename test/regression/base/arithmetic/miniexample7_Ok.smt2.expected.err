terminate called after throwing an instance of 'opensmt::LANonLinearException'
  what():  Term (+ (div 10 y) (* -1 x)) is non-linear
Aborted
