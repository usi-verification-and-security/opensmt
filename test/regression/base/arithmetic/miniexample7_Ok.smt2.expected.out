Term (div 10 y) is non-linear
unknown
